docs/index.html

```diff
@@ -0,0 +1,89 @@
+---
+layout: index
+---
+
+<section class="bg-dark py-3">
+    <div class="container">
+        <div class="text-center">
+            <h1>Collaborate on Code</h1>
+            <h2>Distributed Collaborative Development in your IDE</h2>
+        </div>
+        <div class="row mt-4 mb-1">
+            <div class="mx-auto">
+                <a href="{{ '/documentation/getting-started.html' | absolute_url }}"><button type="button"
+                        class="btn btn-getting-started btn-lg">Getting Started</button></a>
+            </div>
+        </div>
+        <div class="row my-5">
+            <div class="mx-auto">
+                <img class="img-fluid" src="{{ '/assets/images/animation/saros.gif' | relative_url }}" />
+            </div>
+        </div>
+    </div>
+</section>
+
+<section class="my-5">
+    <div class="container">
+        <div class="row py-5 justify-content-center">
+            <div class="col-12 col-sd-10 col-md-8 col-xl-6 mx-2 text-center">
+                <h2 class="mb-3">Several Concurrent Writers</h2>
+                <p class="lead">Share a project in a group of <span class="font-weight-bold">up to five team
```

Comment:
I know I tried to disambiguate the sentence, but it might be more confusing.
I think I should remove the numeric restriction in the sentence and just say:
"Share a project with multiple of your team members in one session. Each member can edit the shared files at the same time"?

Reply:
But, currently, five session participants is the limit (due to the limited color choices; you could argue that 6 is the limit including the default color but whatever). This should still be mentioned on the page for now in my opinion as it is an important restriction of the feature set. Once the color logic has been overhauled and this restriction no longer applies, we can introduce a more general wording.

How about "Share a project with up to other developers in one session. Each member can edit the shared files at the same time."?